packages/terra-section-header/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-section-header",
+  "main": "lib/SectionHeader.js",
+  "version": "0.0.0",
+  "description": "Interactive header component that provides a status indicator.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-section-header",
+    "SectionHeader",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^2.1.0"
+  },
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-arrange": "^2.1.0",
+    "terra-base": "^3.1.0",
+    "terra-mixins": "^1.13.0"
```

Comment:
terra-mixins should also be a peer dependency.

Reply:
We don't actually want to include terra-mixins as both a dependency and peer dependency. This is setup correctly.

Reply:
Numerous other packages list mixins as both a dependency and peer dependency (card, base, badge, status-view etc...). Is there a reason why we wouldn't do it here?

Reply:
Added 0cff014cb138b5ea28c281a727e3eb1e202c5fc4